src/stores/sessions.js

```diff
@@ -36,6 +40,56 @@ export const useSessionsStore = defineStore('sessions', {
           }
         }
       }
+    },
+    async fetchToken () {
+      const requestBody = JSON.stringify(this.currentSession)
+      const response = await fetchApiCall({
+        url: 'http://rti-bridge-dev.lco.gtn/login',
+        method: 'POST',
+        body: JSON.parse(requestBody)
```

Comment:
Nitpick, but it becomes quite hard to read this code when we're JSON string-ifying it and then JSON parsing it before sending it to the API. There must be a way within the this.currentSession computed property so you don't need to do this and can just return a JSON object.

Reply:
would it be okay if the getter stringified and parsed it?